### DISCUSSION TITLE
is there any way to see how things are stored in my memory in zed

### DISCUSSION THREAD

**Discussion by tusharhqq** [zed-industries/zed#14448] [Feature Requests]
below i have added one image of how memory and memory allocations look like, can i see something similar in zed or terminal in any way
![image](https://github.com/user-attachments/assets/83230f06-b96c-457e-a16b-af7fb2890718)

**Comment by SomeoneToIgnore** [comment]
Nope, Zed does not have this functionality yet, unfortunately, not even a hex editor/browser to start with.

**Reply by tusharhqq** [reply]
is there any plan for adding this functionality in 2024?

**Reply by SomeoneToIgnore** [reply]
Not aware of any, but maybe a great contributor comes and adds one.
